apps/homepage/__tests__/e2e/localization.cy.tsx

```diff
@@ -0,0 +1,32 @@
+describe('Localization Tests', () => {
+  const languages = {
+    en: {
+      title: 'HomePage',
+      header: 'Header'
+    },
+    es: {
+      title: 'Pagina Principal',
+      header: 'Encabezado'
+    }
+  };
+
+  Object.keys(languages).forEach(lang => {
+    context(`Language: ${lang}`, () => {
+      beforeEach(() => {
+        cy.visit('/', {
+          headers: {
+            'Accept-Language': lang
```

Comment:
Add another test where no header is present. The default language should be opened.